pom.xml

```diff
@@ -16,8 +16,14 @@
     <description>mmotodo</description>
     <properties>
         <java.version>21</java.version>
+
+            <flyway.url>jdbc:mysql://localhost:3306/mmotodo</flyway.url>
+            <flyway.user>root</flyway.user>
+            <flyway.password>root</flyway.password>
+
```

Comment:
This is probably not code that should be in a production environment

Reply:
If it's needed for the maven flyway plugin to work we can always reference environment variables in our pom.xml but do we need it at all?
${env.FLYWAY_URL}

Reply:
> This is probably not code that should be in a production environment

That's right. I am going to take that out